README.md

```diff
@@ -1,63 +1,39 @@
 # firecrown: the "c" is for "cosmology"
 
-[![CircleCI](https://circleci.com/gh/LSSTDESC/firecrown/tree/master.svg?style=svg)](https://circleci.com/gh/LSSTDESC/firecrown/tree/master)
-
 ## Introduction
+ 
+Firecrown is a Python package that provides the DESC *framework* to implement likelihoods,
+as well as specific likelihood implementations. Firecrown is intended to be usable *from*
+external statistical analysis tools.
 
-Firecrown is most often used along with a *sampler*. There are two samplers
-currently supported:
+Currently, it supports both Cobaya and CosmoSIS, providing the necessary classes or modules
+to allow users of Cobaya or CosmoSIS to call any Firecrown likelihood from within those
+samplers.
 
 * [Cobaya](https://github.com/CobayaSampler/cobaya)
 * [CosmoSIS](https://github.com/joezuntz/cosmosis)
 
-It can also be used as a library in other contexts, and so installation of
+It can also be used as a library in other contexts, and so the installation of
 Firecrown does not *require* the installation of a sampler.
 
 ## Installation Quickstart
 
 > Warning
 >
-> The conda packaging of Firecrown is not yet done.
+> The conda packaging of Firecrown is not yet completed.
 > The following instructions show how we intend the installation to work;
 > however, for now please follow the "Installation of dependencies for development"
 > instructions below.
 
 The easiest way to get started is by using conda. We recommend creating a conda
 environment for your use.
 
-There are different prescriptions, depending upon the choice you make on the use
-of samplers. You can use whatever environment name you prefer for any
-installation.
-
-For Firecrown alone:
+This will install Firecrown as well as the sampler that are currently supported.
```

Comment:
```suggestion
This will install Firecrown as well as the samplers that are currently supported.
```

Reply:
Thanks for the review. I'll update the PR to reflect your fixes.